hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/impl/AbstractFSBuilderImpl.java

```diff
@@ -46,7 +46,7 @@
  * <code>
  *   .opt("foofs:option.a", true)
  *   .opt("foofs:option.b", "value")
- *   .opt("barfs:cache", true)
+ *   .opt("fs.s3a.open.option.etag", "9fe4c37c25b")
```

Comment:
Why isn't etag also a standard option?  The less fs specific stuff the better, right?

Reply:
That's just an example of a non-standard one. No actual s3a support for this as it complicates life too much